CMakeLists.txt

```
@@ -51,6 +51,10 @@ else ()
    message(STATUS "BRAHMA_LIBDIR set to ${BRAHMA_LIBDIR}")
 endif ()
 
+include(${CMAKE_CURRENT_SOURCE_DIR}/cmake/modules/brahma-utils.cmake)
```

```suggestion
include(brahma-utils.cmake)
```

We have the path added to modules already [here](https://github.com/hariharan-devarajan/brahma/blob/develop/CMakeLists.txt#L60C1-L60C73).

---

This doesn't work as you explained, not sure why. Kept as is for now.